### ISSUE TITLE
Questions about point cloud datasets and training overfitting

### ISSUE THREAD

**Issue by laodouza** [hzxie/Stereo-3D-Reconstruction#5]
Hello, Haozhe, first of all congratulations for a great job, and secondly I would like to ask two questions I encountered during the reproduction process (I used the reconstruction point cloud part):
1. I downloaded the dataset you recommended, but there is no point cloud ground truth in it. Where should I download the relevant data?
2. After I used my existing point cloud ground truth data, due to some data processing problems, I only trained the aircraft data set separately, but there will always be overfitting phenomenon. I would like to ask if you have encountered such a problem? ? I put the loss function curve below.
![931669784063_ pic](https://user-images.githubusercontent.com/73707267/205481500-434bfb2f-880c-467a-9154-62b838c659d9.jpg)

**Comment by hzxie** [comment]
1. I forgot to release the point cloud dataset. But you can generate it by yourself. For detailed information, please refer to #3.
2. I cannot remember it clearly (as the work was done in 2018). Maybe you can add more data augmentation to alleviate this phenomenon.
